src/promptflow/promptflow/__init__.py

```diff
@@ -25,5 +25,7 @@
     "load_run",
     "log_metric",
     "ToolProvider",
+    "InputSettings",
```

Comment:
I feel we should not put it in root namesapce. 

promptflow.entities.InputSettings?

Reply:
maybe put it in promptflow.contracts.tool

Reply:
move it to promptflow.entities